src/System.IO.Abstractions.Extensions/IFileSystemExtensions.cs

```diff
@@ -11,5 +11,131 @@ public static IDirectoryInfo CurrentDirectory(this IFileSystem fileSystem)
         {
             return fileSystem.DirectoryInfo.New(fileSystem.Directory.GetCurrentDirectory());
         }
+
```

Comment:
can you please add a
```
#pragma warning( disable: S3447 )
```
at the beginning of this file in order to get rid of the codacity errors
maybe also add a coment why we are doing that

Reply:
Fixed! Note that I had to disable S3874; S3447 is a rule variant specific to parameters marked `[Optional]`.